x-pack/plugins/synthetics/common/rules/alert_actions.test.ts

```diff
@@ -406,7 +406,7 @@ describe('Alert Actions factory', () => {
             eventAction: 'trigger',
             severity: 'error',
             summary:
-              'Monitor {{context.monitorName}} with url {{{context.monitorUrl}}} from {{context.observerLocation}} {{{context.statusMessage}}} The latest error message is {{{context.latestErrorMessage}}}',
+              '{{context.monitorName}} {{context.monitorType}} monitor is {{{context.status}}} from {{context.locationName}} at {{context.checkedAt}}.\nThe error received is: "{{{context.lastErrorMessage}}}".\n{{{context.linkMessage}}}',
```

Comment:
Instead of saying `from {location}` should I do something more like `{{context.monitorName}} ({{context.locationName}})...`

Also should I put the monitor name in quotes?